lib/resources/example_json/r4_example_patient.rb

```diff
@@ -0,0 +1,338 @@
+module Cerner
+  module Resources
+
+    R4_COMBINED_PATIENT_ENTRY ||= {
+      "resourceType": "Patient",
+      "id": "4860007",
+      "meta": {
+        "versionId": "0"
+      },
+      "active": false,
+      "link": [
+        {
+          "other": {
+            "reference": "Patient/4342008"
+          },
+          "type": "replace"
+        }
+      ]
+    }
+
+    R4_PATIENT_ENTRY ||= {
+      "resourceType": "Patient",
+      "id": "4342009",
+      "meta": {
+        "versionId": "43",
+        "lastUpdated": "2019-03-27T15:51:22.000Z"
+      },
+      "text": {
+        "status": "generated",
+        "div": "&lt;div&gt;&lt;p&gt;&lt;b&gt;Patient&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: Active&lt;/p&gt;&lt;p&gt;&lt;b&gt;Name&lt;/b&gt;: SMART, NANCY&lt;/p&gt;&lt;p&gt;&lt;b&gt;DOB&lt;/b&gt;: Aug 11, 1980&lt;/p&gt;&lt;p&gt;&lt;b&gt;Administrative Gender&lt;/b&gt;: Female&lt;/p&gt;&lt;p&gt;&lt;b&gt;Marital Status&lt;/b&gt;: Married&lt;/p&gt;&lt;/div&gt;"
+      },
+      "extension": [{
```

Comment:
the site renders this appropriately but I would prefer if we send this through a json formatter here as well

Reply:
<img width="1175" alt="Screen Shot 2019-03-28 at 10 25 53 AM" src="https://user-images.githubusercontent.com/42553712/55170073-e9e0e700-5143-11e9-8706-3f2152378273.png">

Reply:
right, the site renders it regardless of what spacing we have here. I would prefer the raw json to match the formatted json

Reply:
Adjusted the brackets: https://github.com/cerner/fhir.cerner.com/pull/241/commits/830d1fd3a082e2e565f0cd28b38c79997d97c597

Reply:
907614b

Reply:
https://github.com/cerner/fhir.cerner.com/pull/241/commits/907614b11e96ebb5bd585832252c24e1b8c16d8d